.template/lib/template/application.rb

```diff
@@ -0,0 +1,26 @@
+module Template
+  class Application
+    attr_reader :app_name
+
+    def initialize(app_name = ENV['APP_NAME'])
+      @app_name = app_name
+    end
+
+    # Transform the app name from slug to human-readable name e.g. nimble-web -> Nimble
+    def humanized_name
+      @humanized_name = app_name
+                          .split(/[-_]/)
+                          .map(&:capitalize)
+                          .join(' ')
+                          .gsub(/ Web$/, '')
```

Comment:
`Web` could be stored as a constant:

```suggestion
                          .gsub(/ APP_NAME_SUFFIX$/, '')
```